back/libs/core/src/core.module.ts

```diff
@@ -113,7 +110,7 @@ export class CoreModule {
       exports: [
         CoreAccountService,
         CoreOidcProviderConfigAppService,
-        trackingModule,
+        TrackingModule,
```

Comment:
Not sure if we still need to export the TrackingModule now, since it's just the standard module without any injection.

Reply:
this is deleted in #238